const login = () => {
    try{
        
    }catch(err){
        console.log(err)
    }
}

export default login;